docs_src/OVMS/capiPipelineRun.md

```diff
@@ -28,23 +28,23 @@ Here is the config file location: `configs/opencv-ovms/models/2022/config_templa
     `shape` is optional and takes precedence over batch_size, please remove this attribute if you don't know the value for the model.
 
 !!! Note
-    Please leave `target_device` value as it is, as the value `{target_device}` will be recognized and filled out by script run.
+    Please leave `target_device` value as it is, as the value `{target_device}` will be recognized and replaced by script run.
 
-You can find each parameter description from model server [ovms_docs_parameters](https://docs.openvino.ai/2023.1/ovms_docs_parameters.html).
+You can find the parameter description from model server [ovms_docs_parameters](https://docs.openvino.ai/2023.1/ovms_docs_parameters.html).
 
 ## Add Environment Variable File
-You can add environment variable file to `configs/opencv-ovms/envs/` directory and you can add more than one environment variable file for your pipeline. For face detection pipeline run, we have added `configs/opencv-ovms/envs/capi_face_detection.env`, below is a list of explanation for each environment variable and current default values we set for face detection pipeline run, this list can be extended for any future modification.
+You can add mulitple environment variable files to `configs/opencv-ovms/envs/` directory for your pipeline. For face detection pipeline run, we have added `configs/opencv-ovms/envs/capi_face_detection.env` environment variable file. Below is a list of explanation for all environment variables and current default values we set for face detection pipeline run, this list can be extended for any future modification.
 
 | EV Name                   | Description                                           |
 | --------------------------| ------------------------------------------------------|
-| RENDER_PORTRAIT_MODE      | rendering in portrait mode                            |
-| GST_DEBUG                 | GStreamer debug mode option                           |
-| USE_ONEVPL                | using OneVPL CPU & GPU Support                        |
+| RENDER_PORTRAIT_MODE      | value 1 for rendering in portrait mode                |
```

Comment:
Can we split this table into 3 columns to make it easier to read? 

| EV Name   | Default Value   | Description   |